Source/Types/Script.swift

```diff
@@ -9,18 +9,17 @@
 import Foundation
 
 public struct Script: Codable, Param {
-    public let version: UInt8
     public let args: [HexString]
     public var binaryHash: H256
 
     enum CodingKeys: String, CodingKey {
-        case version, args
+        case args
         case binaryHash = "binary_hash"
     }
 
     static let alwaysSuccessHash: H256 = "0000000000000000000000000000000000000000000000000000000000000001"
```

Comment:
Does it has `0x` prefix ?

Reply:
Good question. The type hash is not affected whether we have `0x` or not. But I need to check if `codeHash` should always have `0x` to keep it consistent. (adding a new issue for `v0.11.0` or later)